backend/src/middleware/notifications/notificationsMiddleware.js

```diff
@@ -1,6 +1,8 @@
 import extractMentionedUsers from './mentions/extractMentionedUsers'
 import { validateNotifyUsers } from '../validation/validationMiddleware'
 
+const debug = require('debug')('backend:notificationsMiddleware')
```

Comment:
```suggestion
const debug = require('debug')('human-connection-backend:notificationsMiddleware')
```
This should be the `name` in our `package.json` as per [convention](https://www.npmjs.com/package/debug#conventions). It's not like this everywhere in the code base because we recently changed the `name` (could be refactored, too).

Reply:
Ah okay. @roschaefer 
Have saved that as a reminder. May I will create an issue …

Reply:
Done by the way 👍🏼